src/stylesheets/components/_dsd24-banner.scss

```diff
@@ -0,0 +1,18 @@
+.app-dsd24-banner {
```

Comment:
I'm not sure about making components for a specific year, it feels a bit messy.  Could the commit, or the component comments include some rationale for this choice?

Alternatively, could we genericise the classnames? Like `app-banner-with-image` or something. And `app-call-to-action`, etc. That way we can hopefully just tweak these components for future events.

Reply:
Have removed `dsd24` prefix from all macros and associated styles. Kept the `dsd24` prefix for the classes I had to apply to take precedence over the existing styles in the application (and don't seem to have much immediate use out of the pages).